crates/modelardb_server/src/storage/compressed_data_buffer.rs

```diff
@@ -24,14 +24,14 @@ use std::sync::Arc;
 use datafusion::arrow::compute;
 use datafusion::arrow::record_batch::RecordBatch;
 use datafusion::parquet::format::SortingColumn;
+use modelardb_common::metadata::compressed_file::CompressedFile;
 use modelardb_common::metadata::model_table_metadata::ModelTableMetadata;
+use modelardb_common::metadata::univariate_id_to_column_index;
```

Comment:
Maybe only import `metadata` and use the function as `metadata::univariate_id_to_column_index()` for consistency.

Reply:
Fixed.